src/ripple/app/misc/ValidatorList.h

```diff
@@ -505,6 +505,50 @@ class ValidatorList
         return {quorum_, trustedSigningKeys_};
     }
 
+    /**
+     * get the trusted master public keys
+     * @return the public keys
+     */
+    hash_set<PublicKey>
+    getTrustedMasterKeys()
```

Comment:
Nit: In general we try to not to define function implementations in the class definition (it makes it harder to read the interface). These are all very small functions, so it's not a big deal, but I'd probably more them to the cpp file.

Reply:
Right, moved to .cpp file.